### PR TITLE
Use new coding standard

### PR DESCRIPTION
**Pull request by Koopzington**

### CODE REVIEW

**Comment by xtreamwayz** [comment]
I'm having my doubts if this should be included in a skeleton. It adds a lot of ZF specific rules (especially with the upcoming changes). I prefer a simple rule set close to PSR-2.

**Comment by Koopzington** [comment]
that's why it should get removed after the installation if i did everything right :D

**Comment by xtreamwayz** [comment]
Oh yeah it does :)

Only `phpcs.xml` is left behind.

**Comment by weierophinney** [comment]
@Koopzington The only problem, however, is that the `phpcs.xml` file is left in an unusable state after install, as it's referencing the `zend-conding-standard.xml` file...

The solution would be to have a `phpcs.xml.dist` file with the non-ZF ruleset, and remove the `phpcs.xml` file following install.

File: src/ExpressiveInstaller/OptionalPackages.php [hunk 1]
```diff
@@ -240,6 +240,7 @@ private static function cleanUp(IOInterface $io, $projectRoot)
         unlink($projectRoot . '/CHANGELOG.md');
         unlink($projectRoot . '/CONDUCT.md');
         unlink($projectRoot . '/CONTRIBUTING.md');
+        unlink($projectRoot . '/phpcs.xml');
```
Comment (weierophinney) [inline src/ExpressiveInstaller/OptionalPackages.php]: Okay, missed that you were already removing it. However, we need a standard in place when done.

I'm testing the following locally:

- Including a `phpcs.xml.dist` file with the original PSR-2 ruleset.
- Adding a devleopment requirement for `"squizlabs/php_codesniffer": "^2.7"`

This combination should present a usable, minimal phpcs toolchain from the outset, which developers can then customize.

**Comment by weierophinney** [comment]
Merged to develop for release with 1.1, and includes the changes I indicated above.

**Pull request closed without merging**